ansible/inventory-check.yml

```diff
@@ -129,7 +129,7 @@
 
     - name: stamp the inventory that was checked
       assert:
-        that: "'{{ inventory_hash }}' == '0aeedabeac0fcdf921b952ead8f920b847e7041aae62d7cc52d52184210aea3c'"
+        that: "'{{ inventory_hash }}' == '390072320b73bf063d6602bfb4545329099cb8e6d1487a26b6d4c28f7dbc7aae'"
```

Comment:
How is it possible that the check passed? It failed due to #276 for me.

Reply:
Not sure. However the inventory check was failing in master as well.